docs/meetings/2023/03-March/README.md

```diff
@@ -1,4 +1,4 @@
 # March, 2023
 
-* Liang Yan: [Stable Diffusion: ChatGPT in Painting](Stable-Diffusion-v2.pdf)
+* Liang Yan: [Stable Diffusion: ChatGPT in Painting](https://github.com/KYOSS/website/blob/main/docs/meetings/2023/03-March/Stable-Diffusion-v2.pdf)
```

Comment:
I think you can just do absolute relative to the docroot.  For example, look at this:

https://github.com/KYOSS/website/blame/main/docs/meetings/2019/07-July/README.md#L7

Reply:
nice. the root path of the website is under /docs,  good to know. Changed.